bin/generate_spec.rb

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env ruby
+
+require 'pathname'
+
+spec_dir = Pathname.new("../rpm_spec").expand_path(__dir__)
+manageiq_spec = spec_dir.join("manageiq.spec.in").read
+
+Dir.glob("#{spec_dir.join("subpackages/*")}") do |spec|
```

Comment:
Minor, but you might want to do `Dir.glob().sort` so that you get consistent output.